config/settings.defaults.js

```diff
@@ -17,14 +17,14 @@ module.exports = {
   },
 
   mongo: {
-    url: `mongodb://${process.env['MONGO_HOST'] || 'localhost'}/sharelatex`
+    url: process.env['MONGO_CONNECTION_STRING'] || `mongodb://${process.env['MONGO_HOST'] || 'localhost'}/sharelatex`
   },
 
   redis: {
     web: {
       host: process.env['REDIS_HOST'] || 'localhost',
       port: '6379',
-      password: ''
+      password: process.env['REDIS_PASSWORD'] || ''
```

Comment:
how were we getting the password in before, or was there none?

Reply:
we were using the settings files in chef.